scripts/vulnerabilities.sh

```diff
@@ -50,39 +39,34 @@ find_askpass_image_tag() {
 
 GCENODE_ASKPASS_TAG="${GCENODE_ASKPASS_TAG:-$(find_askpass_image_tag)}"
 
-# TODO: share with Makefile to keep in sync
-configsync_images=(
-  "gcr.io/${GCP_PROJECT}/reconciler"
-  "gcr.io/${GCP_PROJECT}/reconciler-manager"
-  "gcr.io/${GCP_PROJECT}/admission-webhook"
-  "gcr.io/${GCP_PROJECT}/hydration-controller"
-  "gcr.io/${GCP_PROJECT}/hydration-controller-with-shell"
-  "gcr.io/${GCP_PROJECT}/oci-sync"
-  "gcr.io/${GCP_PROJECT}/helm-sync"
-  "gcr.io/${GCP_PROJECT}/nomos"
-)
-
 # Build a full list of images with tags
 images=(
-  "gcr.io/${GCP_PROJECT}/gcenode-askpass-sidecar:${GCENODE_ASKPASS_TAG}"
+  "gcr.io/config-management-release/gcenode-askpass-sidecar:${GCENODE_ASKPASS_TAG}"
 )
 
 # find the "name:tag" of images in the manifests directory.
 find_manifest_images() {
-  grep -rohE --color=never "image:\s+\S+:\S+" manifests/**/*.yaml |
+  grep -rohE --color=never "image:\s+\S+:\S+" .output/staging/**/*.yaml |
```

Comment:
Can you add a check that validates this dir and/or yaml files exist? It should prompt you to render or download the manifests with the make targets to call.

It should probably also print something up front to remind you that it's extracting images from local files at this location, in case you forget and run it on stale manifests without re-rendering.

Reply:
Added a check and some extra logging